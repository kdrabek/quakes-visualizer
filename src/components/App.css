body {
  margin: 0;
  font-family: 'Montserrat', sans-serif;
}

.grid {
  display: grid;
  height: 100vh;
  grid-template-rows: 1fr 9fr;
  grid-template-columns: 1fr 1fr;
}

