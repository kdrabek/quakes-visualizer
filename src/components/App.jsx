import React, { PureComponent } from 'react';
import './App.css';

import Title from './Title';
import Options from './Options';
import EarthquakesList from './EarthquakesList';


export default class App extends PureComponent {
  state = {
  };

  render() {
    return (
      <div className="wrapper">
        <Title />
        <Options />

        <div className="list">
          <EarthquakesList />
        </div>
        <div className="map">2</div>
      </div>
    );
  }
}
