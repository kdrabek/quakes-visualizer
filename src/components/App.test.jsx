import { shallow } from 'enzyme';
import App from './App';


describe('App component', () => {
  const props = {};

  it('should match snapshot', () => {
    const wrapper = shallow(<App {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
