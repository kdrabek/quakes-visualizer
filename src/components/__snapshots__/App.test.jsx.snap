// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match snapshot 1`] = `
<div
  className="grid"
>
  <Title
    text="earthquake browser"
    url="https://gitlab.com/kdrabek/earthquakes-visualizer"
  />
  <Connect(OptionsContainer) />
  <Connect(Earthquakes) />
  <Connect(EarthquakesMap) />
</div>
`;
